Geospatial.md

```diff
@@ -162,3 +195,19 @@ The axis order of the coordinates in WKB and bounding box stored in Parquet
 follows the de facto standard for axis order in WKB and is therefore always
 (x, y) where x is easting or longitude and y is northing or latitude. This
 ordering explicitly overrides the axis order as specified in the CRS.
+
+# Invalid geospatial values
+
+An invalid geospatial value refers to the coordinate values of a non-`null` 
+geospatial instance that are encoded in a valid WKB format, but are not 
```

Comment:
As we have mentioned `a valid WKB format`, do we need to provide guidelines for `invalid WKB format`?

Reply:
I think it’s very difficult to enumerate all possible invalid WKB formats, and chasing this could easily become another rabbit hole. Since the WKB standard clearly defines what a valid format is, I suggest we leave this out.

Reply:
Agreed. Is there any well-known reference for this?

Reply:
I think we already provided references to the WKB standard?